curriculum/challenges/english/01-responsive-web-design/basic-css/size-your-images.english.md

```diff
@@ -34,8 +33,8 @@ Create a class called <code>smaller-image</code> and use it to resize the image
 tests:
   - text: Your <code>img</code> element should have the class <code>smaller-image</code>.
     testString: assert($("img[src='https://bit.ly/fcc-relaxing-cat']").attr('class') === "smaller-image");
-  - text: Your image should be 100 pixels wide. Browser zoom should be at 100%.
-    testString: assert($("img").width() === 100);
+  - text: Your image should be 100 pixels wide.
+    testString: assert($("img").width() < 200 && code.match(/(?<=<style>([\s\S])*)\.smaller-image[\s\n]*{([\s\S]*;)?[\s\n]*width\s*:\s*100px\s*(?=[\s\S]*<\/style>)/i));
```

Comment:
@Sky020 Using an example of something @moT01 used in another challenge, the following regex is flexible enough to allow the use to add additional properties before and after the `width: 100px;` and even allow not using the semi-colon if it is the last property/value pair in the selector definition.
```suggestion
    testString: assert($("img").width() < 200 && code.match(/(?<=<style>([\s\S])*)\.smaller-image\s*{([\s\S]*;)?\s*width\s*:\s*100px\s*(;|})[\s\S]*(?=<\/style>)/i));
```